webapi/src/main/scala/org/knora/webapi/util/PermissionUtilADM.scala

```diff
@@ -22,120 +22,103 @@ package org.knora.webapi.util
 import com.typesafe.scalalogging.Logger
 import org.knora.webapi.messages.admin.responder.permissionsmessages.PermissionType.PermissionType
 import org.knora.webapi.messages.admin.responder.permissionsmessages.{PermissionADM, PermissionType}
+import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
 import org.knora.webapi.messages.v1.responder.usermessages.UserProfileV1
 import org.knora.webapi.responders.v1.GroupedProps.{ValueLiterals, ValueProps}
 import org.knora.webapi.{IRI, InconsistentTriplestoreDataException, OntologyConstants}
 import org.slf4j.LoggerFactory
 
-import scala.collection.immutable.Iterable
 
 /**
-  * A utility that responder actors use to determine a user's permissions on an RDF subject in the triplestore.
+  * A utility that responder actors use to determine a user's permissions on an RDF entity in the triplestore.
   */
 object PermissionUtilADM {
 
-    val log = Logger(LoggerFactory.getLogger(this.getClass))
```

Comment:
Any particular reason why you would remove the logger? If we wont to filter logging output for this class, then it needs to be defined here.

Reply:
It was unused. I can put it back if you like.

Reply:
ah, found it. wasn't removed. just moved.